cli/command/service/list.go

```diff
@@ -40,6 +36,8 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&opts.quiet, "quiet", "q", false, "Only display IDs")
+	flags.BoolVar(&opts.noTrunc, "no-trunc", false, "Do not truncate the output")
```

Comment:
I wonder if we should consider introducing `ShortID` and/or `TruncID` placeholders. The `TruncID` placeholder has the behavior of what `ID` has now (so, truncated by default, but show full ID if `--trunc` is added. That way people can define a template where the output _always_ has the full-id, or has an ID that can be truncated or full.